lib/livebook_web/live/session_live/standalone_runtime_component.ex

```diff
@@ -20,22 +59,60 @@ defmodule LivebookWeb.SessionLive.StandaloneRuntimeComponent do
         Start a new local Elixir node to evaluate code. Whenever you reconnect this runtime,
         a fresh node is started.
       </p>
-      <.button phx-click="init" phx-target={@myself} disabled={@runtime_status == :connecting}>
-        <%= label(@runtime, @runtime_status) %>
-      </.button>
+      <.form
+        :let={f}
+        for={@changeset}
+        as={:data}
+        phx-submit="init"
+        phx-change="validate"
+        phx-target={@myself}
+        autocomplete="off"
+        spellcheck="false"
+      >
+        <div class="flex flex-col space-y-4 mb-5">
+          <.text_field field={f[:erl_flags]} label="Erl flags" />
```

Comment:
We need to have something that indicates people do not have to fill in or worry about his field. If we could hide it behind a `<details>` saying "Advanced configuration", it would be even better, but if not, the bellow should do it:

```suggestion
          <.text_field field={f[:erl_flags]} label="Erl flags (advanced configuration)" />
```

Reply:
<img width="921" alt="image" src="https://github.com/user-attachments/assets/e0f45d3a-e624-4c3a-8b18-25e46ccf6fb6">

<img width="915" alt="image" src="https://github.com/user-attachments/assets/0ce1487e-812a-44b3-861a-1fc9d3799655">